finals-2024-01-16/code/5040-dara-poposka/src/app/country-detail-component/country-detail-component.component.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
We definitely don't have anything resembling to this data in the back-end :smile: